### ISSUE TITLE
Working of workflows on Scalable environment 

### ISSUE THREAD

**Issue by galipcevrik** [danielgerlag/workflow-core#755]
Hi @danielgerlag 

We use the workflow-core project in our project with the help of an interface, and this project runs on a scalable environment on the cloud. We also use redis as the provider for workflow-core.

We use the related workflows in json format and before starting them, we load them with the DefinationLoad method. The workflows we have installed are working, but since there are more than one container and in-memory, it constantly prints the error stating that the relevant workflow is not registered.

To give an example, there are 3 containers. When I start a workflow on the application, it is working properly on a continer and it seems to be working on the other 2 containers, and it throws an error saying that it is not registered because defination json is not installed on these containers.

Server-based management seems necessary here. As an example, I can mention the server structure on Hangfire.

Do you plan to make a development on this issue? Or what should I do to avoid these mistakes?

thanks good work.

**Comment by danielgerlag** [comment]
Workflow core currently does not have a backplane for synchronizing definitions across node... the Conductor project has a redis implementation for this - https://github.com/danielgerlag/conductor/blob/master/src/Conductor.Domain/Services/RedisBackplane.cs
Do you think there would be value in adding this feature to workflow core too?

**Comment by galipcevrik** [comment]
Sorry for our late reply.

 I think it will be a very useful development.

**Comment by wfcore-manager** [comment]
Closing as not planned — this has been open for several years without recent progress. Please open a new issue if this is still relevant on a current Workflow Core release.
